build/compile/Microsoft.DotNet.Cli.LzmaArchive.targets

```diff
@@ -14,7 +14,7 @@
             DependsOnTargets="SetupNuGetPackagesArchiveInputsOutputs;
                               SetupAzureBlobInformation;
                               GenerateNuGetPackagesArchive"
-            Condition=" '$(UploadNuGetPackagesArchiveToAzure)' == 'true' And  '$(PUBLISH_TO_AZURE_BLOB)' != '' ">
+            Condition=" '$(UploadNuGetPackagesArchiveToAzure)' == 'true' And  '$(PUBLISH_LZMA_TO_AZURE_BLOB)' != '' ">
```

Comment:
Yes, this is so that only one of our VSO build definitions (OSX) will upload the archive.